Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1100,6 +1106,7 @@ - (BNCLinkData *)prepareLinkDataFor:(NSArray *)tags andAlias:(NSString *)alias a
     [post setupStage:stage];
     [post setupAlias:alias];
     [post setupMatchDuration:duration];
+    if (ignoreFirstClick) [post setupIgnoreFirstClick:ignoreFirstClick];
```

Comment:
Note that the other items here have the `if` within the setup method itself. This should be move there for consistency. Also, I really prefer bracketed `if`s, even for one liners like this.